Rebuild-Black-Business/RBB-Website
# [fix] Fix issue with Chakra UI theme config

# Describe your PR
Prior to this PR, our custom values in the Chakra UI theme config were not being properly provided to our application. To fix this, I:

1. Removed the `wrapRootElement` API calls in `gatsby-ssr.js` and `gatsby-browser.js`
2. Moved `customTheme` object into `src/gatsby-plugin-chakra-ui/theme.js`

## Pages/Interfaces that will change
This will effect all pages and components within the app.

## Steps to test

1. Import `useTheme` in a component and add `const customTheme = useTheme()`
2. Log out `customTheme` to confirm our custom values from `theme.js` are in the object

### Additional notes
